src/core/src/core_logic/PatchInstaller.py

```diff
@@ -79,6 +79,7 @@ def start_installation(self, simulate=False):
 
         if self.execution_config.max_patch_publish_date != str():
             self.package_manager.set_max_patch_publish_date(self.execution_config.max_patch_publish_date)
+            # todo: validate if this meets the Az Linux and tdnf requirements
```

Comment:
Marking todo comment spot for PR evolution.

Reply:
Removed